packages/tools/fetch-tool/package.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "@fluid-tools/fetch-tool",
   "version": "0.59.3000",
+  "private": true,
```

Comment:
Feels like a potential loss not to publish some tool packages.  Can we remove this namespace from both lists in the policy check so they can be classified on a case by case basis?

Reply:
Like you said, layer check should protect us here.  No layer says they may depend on Tools layer (If I'm reading layerInfo.json properly... might be good to test it out real quick)

Reply:
Oh yikes, we didn't use to publish `@fluid-tools` AFAIK, not sure when that started.  I don't think we've audited those packages really, so not sure if they're really in a state where we should feel comfortable with external customers using them...  In particular I don't think we have any kind of support plan for them if a customer hits an issue necessarily.

Reply:
> Oh yikes, we didn't use to publish @fluid-tools AFAIK

Not following...

Reply:
Followed up offline with Mark - I'll leave the `@fluid-tools` alone for now, and make a proposal as to what we should do with them.